### ISSUE TITLE
func_healthcharger and func_recharge aren't functional

### ISSUE THREAD

**Issue by MyGamepedia** [StrataSource/Portal-2-Community-Edition#1381]
### Describe the bug

We have `func_healthcharger` and `func_recharge` in game, but they not functional.

https://github.com/StrataSource/Portal-2-Community-Edition/assets/103366204/d9f36658-55d0-4d4a-b6f3-f65aafd3de4e

### Issue Map

Any with `func_healthcharger` or `func_recharge`.

### To Reproduce

1. Open my map for quick test. [func_chargers_test.zip](https://github.com/StrataSource/Portal-2-Community-Edition/files/11779414/func_chargers_test.zip)
2. Try to use `func_healthcharger` and `func_recharge`.
3. Pay attention that you still have red screen, also turret kill you with the same speed.

### Operating System

Any.

**Comment by ENDERZOMBI102** [comment]
isn't this for the same "they're trying to cast to hl2 player and fail so will not work"?

**Comment by MyGamepedia** [comment]
> isn't this for the same "they're trying to cast to hl2 player and fail so will not work"?

Sure.

**Comment by Luke18033** [comment]
These are the legacy HL1 charger entities, which I think still require you to have the HEV suit unlike the prop health charger. You can try giving the player an `item_suit` and see if that fixes it, though note that suit armor doesn't function so the non-health chargers are basically useless.

**Comment by Luke18033** [comment]
Oh also because of #888 you have to spam +use to get any actually useful charge, you can't just hold it down

**Comment by MyGamepedia** [comment]
> Oh also because of #888 you have to spam +use to get any actually useful charge, you can't just hold it down

I know about it, I spam in the video, but also at start player get 99 heal, so, it's just isn't functional.

**Comment by Luke18033** [comment]
Did you try giving yourself a `item_suit` like I suggested?

**Comment by MyGamepedia** [comment]
> Did you try giving yourself a `item_suit` like I suggested?

Yes, I also pick up suit at map start in the video, it's change nothing. And about suit power - `item_suitcharger` work, but glitched.

**Comment by ertup0** [comment]
func_healthcharger appears to work now
